invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentsSearch/components/ResultsList/ResultsList.js

```diff
@@ -22,7 +22,8 @@ export class ResultsList extends Component {
         'Title',
         'Authors',
         'Series',
-        'Items',
+        'Available Items',
+        'Past Loans',
```

Comment:
Just IMHO: `Past Loans` -> `N. Loans` ?

Reply:
I don't mind, but I kept it like this in order to align with all the other variable names.